### ISSUE TITLE
move Other Understanding Documents contents

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wcag#2879]
ref: https://www.w3.org/WAI/WCAG22/Understanding/#other

I wonder if we can get rid of this list and most of these pages?

* **Introduction to Understanding WCAG** &mdash; I think the content should be in the About Understanding page. Then no link here.
* **Understanding Techniques for WCAG Success Criteria** &mdash;  I think this content should be in the About Techniques page. And it should be linked to in the About Understanding page. Then no link here.
* **Understanding Test Rules for WCAG Success Criteria** &mdash; I think all this content is in [About ACT Rules](https://www.w3.org/WAI/standards-guidelines/act/rules/about/]) Therefore, I think the About Understanding page and About Techniques page should have a short section on "Test Rules" and then link to there. Then no link here.
* **Understanding Conformance &mdash; This is important. It should be in the Page Contents list &mdash; probably at the same level as POUR. see [structure issue](https://github.com/w3c/wcag-redesign/issues/6)**
* **How to Refer to WCAG from Other Document**s &mdash; Is all this detail important? Can we consolidate it into [Referencing and Linking to WAI Guidelines and Technical Documents](https://www.w3.org/WAI/standards-guidelines/linking/)?
* **Documenting Accessibility Support for Uses of a Web Technology** &mdash;  Do we still want this as a separate page listed here?
* **Understanding Metadata** &mdash;  Do we still want this as a separate page listed here?
